.eslintrc.js

```diff
@@ -2,11 +2,15 @@ module.exports = {
   extends: [
     'eslint:recommended',
     'plugin:react/recommended',
-    'plugin:prettier/recommended',
     'airbnb',
+    'plugin:prettier/recommended',
   ],
-  plugins: ['prettier', 'react'],
+  plugins: ['react', 'prettier'],
   rules: {
     'prettier/prettier': ['error'],
+    'react/jsx-filename-extension': [1, { extensions: ['.js'] }],
+  },
+  env: {
+    browser: true,
```

Comment:
Need this set so that eslint understands browser built-in JS objects, e.g. `window`.

![image](https://user-images.githubusercontent.com/833231/92986804-793e9f00-f48b-11ea-995f-745af6e4644e.png)